internal/db/test/test.go

```diff
@@ -43,7 +43,7 @@ func Run(ctx context.Context, testFiles []string, config pgconn.Config, fsys afe
 		return errors.Errorf("failed to resolve absolute path: %w", err)
 	}
 	dstPath := "/tmp"
-	binds := []string{fmt.Sprintf("%s:%s:ro", srcPath, dstPath)}
+	binds := []string{fmt.Sprintf("%s:%s:ro,z", srcPath, dstPath)}
```

Comment:
There's another problem with `z` label for podman on macOS. https://github.com/supabase/cli/pull/2392

I think we have to use a platform specific implementation for this, ie. docker_linux.go

Reply:
Ah ! Nice catch, thanks for the context. I'll create an utils function checking if SELinux is present and "wrapping" the volumes mount with or without the `z` if necessary.

Something like: `GetVolumeBindMode('ro' | 'rw')`